Setup/InstallSchema.php

```diff
@@ -30,87 +30,93 @@ public function install(SchemaSetupInterface $setup, ModuleContextInterface $con
     {
         $installer = $setup;
         $installer->startSetup();
-        // Create transaction data schema
-        $table = $installer->getConnection()->newTable($installer->getTable(TransactionInterface::TABLE_NAME));
-        $table->addColumn(
-            TransactionInterface::ENTITY_ID,
-            Table::TYPE_INTEGER,
-            null,
-            [
-                'identity' => true,
-                'unsigned' => true,
-                'nullable' => false,
-                'primary'  => true
-            ],
-            'id'
-        );
 
-        $table->addColumn(
-            TransactionInterface::ORDER_ID,
-            Table::TYPE_TEXT,
-            255,
-            [],
-            'Order ID'
-        );
+        if ($setup->getConnection()->isTableExists($setup->getTable('sdm_valitor')) == true) {
+            $setup->getConnection()->renameTable($setup->getTable('sdm_valitor'), $setup->getTable('sdm_altapay'));
```

Comment:
Why not using TransactionInterface::TABLE_NAME instead of 'sdm_altapay' ?

Reply:
Updated accordingly.